packages/apalis-redis/src/storage.rs

```diff
@@ -926,9 +924,13 @@ where
     }
 
     /// Re-enqueue some jobs that might be abandoned.
-    pub async fn reenqueue_active(&mut self, job_ids: Vec<&TaskId>) -> Result<(), RedisError> {
+    pub async fn reenqueue_active(
```

Comment:
This is also a breaking change. I am curious, how are you calling this method?

Reply:
I mean if its in a loop? Because this seems to be a bug, the backend doesnt call the method and it should

Reply:
@geofmureithi I'm not calling, this method was broken completely as it used the wrong Redis key. I attempted to fix it by changing the signature.

Many methods in the `RedisStorage` are completely broken, such as `retry()`, which passes wrong arguments to the Lua script.